### ISSUE TITLE
Add a way to comment on blog posts

### ISSUE THREAD

**Issue by joostschouppe** [osmbe/website#82]
We used a neat little system at sotm 2016, I'll look it up

**Comment by jbelien** [comment]
Aouch ... Really not sure it's a good idea :/

I understand why ... but it will require moderation (and could lead to quite a heavy work load).

**Comment by jbelien** [comment]
I don't know what you used for SotM 2016 but I guess it was [Disqus](https://disqus.com/), [JustComments](https://just-comments.com/) or something similar.

**Comment by joostschouppe** [comment]
Yes, we'd need to think about it properly. Maybe we can just do it for select articles? Especially for the latest blog post, I think it might be useful. Though an explicit contact link is also an option of course.

And yes, we used Disqus. I think the moderation queue there is pretty good

**Comment by jbelien** [comment]
Also, if we implement Disqus (or other) to the website, because the free version use ads (and most probably tracking), we will fall under the EU Cookie Policy and we will have to add the whole "Accept the cookie" thing.

I was trying to avoid that :P

**Comment by jbelien** [comment]
Other alternative to maybe have a look at : https://gohugo.io/content-management/comments/#comments-alternatives

**Comment by jbelien** [comment]
https://utteranc.es/ could be a solution too (see https://danyow.net/using-github-issues-for-blog-comments/).

Would even be better to use https://docs.github.com/en/discussions (Utterances doesn't support it - yet ?) !

**Comment by jbelien** [comment]
See https://github.com/laymonage/giscus
